cirq-google/cirq_google/engine/stream_manager.py

```diff
@@ -121,6 +121,11 @@ def __init__(self, grpc_client: quantum.QuantumEngineServiceAsyncClient):
         # interface.
         self._response_demux = ResponseDemux()
         self._next_available_message_id = 0
+        self._executor.submit(self._init_request_queue).result()
+
+    async def _init_request_queue(self) -> None:
+        await asyncio.sleep(0)
```

Comment:
Why this sleep? It's not needed if the only point is to make the function async, since `async def` is enough for that and you don't actually need an `await` (note this is different from generator functions which do need at least one `yield` to be recognized as a generator).